cl/people_db/filters.py

```diff
@@ -199,6 +201,11 @@ class PersonFilter(FilterSet):
         choices=Race.RACES, method="filter_race"
     )
 
+    fullname = filters.Filter(method="filter_fullname")
+
+    def filter_fullname(self, queryset, name, value):
+        return lookup_judge_by_first_or_last_name(queryset, value)
```

Comment:
Last name is indexed in the DB, but not first. I think we'll be OK. (It's a small dataset.)

Reply:
> The disclosure thumbnail is laid out a bit oddly at certain viewport sizes:
> 
> ![Screenshot from 2021-10-04 17-42-31](https://user-images.githubusercontent.com/236970/135942948-25a36cb4-3a3e-4cf4-8cf6-c7108266e8db.png)

yeah - its a weird situation where the vast majority should be vertical but a number of new jefs disclosures are horizontal.

Reply:
√